lib/node_modules/@stdlib/blas/base/drot/test/test.drot.js

```diff
@@ -0,0 +1,576 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Float64Array = require( '@stdlib/array/float64' );
+var dcopy = require( '@stdlib/blas/base/dcopy' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var drot = require( './../lib/drot.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof drot, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function has an arity of 7', function test( t ) {
+	t.strictEqual( drot.length, 7, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function applies a plane rotation with x stride as 1 and y stride as 1', function test( t ) {
+	var xexpected;
+	var yexpected;
+	var nvalues;
+	var xvalues;
+	var yvalues;
+	var delta;
+	var tol;
+	var out;
+	var ex;
+	var ey;
+	var ix;
+	var iy;
+	var c;
+	var j;
+	var k;
+	var x;
+	var y;
+
+	xexpected = [
+		[ 0.6, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, -0.46, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, -0.46, -0.22, 1.06, 0.9, -0.3, -0.4 ]
+	];
+	yexpected = [
+		[ 0.5, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.04, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.04, -0.78, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.04, -0.78, 0.54, 0.08, -0.6, 0.2, 0.8 ]
+	];
+	nvalues = [ 0, 1, 2, 4 ];
+	ix = 1;
+	iy = 1;
```

Comment:
These variables are not necessary. And also this naming is confusing as we don't name strides in this manner; instead, we'd use `sx` and `sy`. Regardless, these values can be inlined below.